packages/standards/biome.jsonc

```diff
@@ -33,17 +33,22 @@
         // be the case. https://biomejs.dev/linter/domains/#project-rules
         "noFloatingPromises": "error",
 
+        "noSecrets": "error",
         "noShadow": "error",
       },
       "performance": {
         "noBarrelFile": "error",
       },
       "style": {
         "noParameterAssign": "error",
+        "useConsistentArrayType": "error",
+        "useDefaultParameterLast": "error",
```

Comment:
Enabled in `@readme/eslint-config/typescript` via [@typescript-eslint/default-param-last](https://typescript-eslint.io/rules/default-param-last).